index.rst

```diff
@@ -52,6 +52,64 @@
 
 .. Add content here.
 .. Do not include the document title (it's automatically added from metadata.yaml).
+.. _create-your-own-technote:
+
+Create Your Own Technote
+========================
+
```

Comment:
```
.. warning
  Make sure that you are a member of the tstn organization before attempting this process otherwise shenanigans will ensue.
  Ask any listed administrators for membership.